app/components/paperShow/components/collectionNoteList.scss

```diff
@@ -9,8 +9,6 @@
     border-radius: 8px;
     color: $black0;
     font-size: 14px;
-    word-break: break-all;
```

Comment:
can u explain to me why these lines are removed?
cuz if there aren't these lines, white spaces(\n, space) aren't rendered properly.